### DISCUSSION TITLE
Zigbee channel 19 utilization is 96.64%! - What utiliuzation means ?

### DISCUSSION THREAD

**Discussion by pipiche38** [zigpy/zigpy#1316] [Q&A] [answered]
Can you clarify what utilisation stand for ? Is that not a level of interferences ?

https://github.com/zigpy/zigpy/blob/e54dab0fb0755d421fe610825c8c72b9542fbfca/zigpy/application.py#L198

**Comment by puddly** [comment, marked as answer]
Energy scans don't distinguish between noise due to 802.15.4 traffic (utilization) or just background noise due to WiFi, etc. I called it utilization because you don't know which one it is.
